cyhy_report/customer/report.mustache

```diff
@@ -590,26 +593,26 @@ As you review the report, you may have additional questions. Check out the answe
         {\RCSourceSansProTwelve <<#calc.address_count_up_flag>>Increase of \textbf{\numprint{<<&calc.address_count_up>>}}<</calc.address_count_up_flag>>
         <<#calc.address_count_down_flag>>Decrease of \textbf{\numprint{<<&calc.address_count_down>>}}<</calc.address_count_down_flag>>
         <<#calc.address_count_flat_flag>>No Change<</calc.address_count_flat_flag>>}
-      \end{minipage}  % END: ADDRESSES OWNED
+      \end{minipage}  % END: ASSETS OWNED
       \begin{minipage}[t]{0.1\textwidth}
          \  % Spacer column
       \end{minipage}
       \begin{minipage}[t]{0.45\textwidth}
-        \textcolor{rc-dark-green}{\RCSourceSansProBoldFourteen ADDRESSES SCANNED}
+        \textcolor{rc-dark-green}{\RCSourceSansProBoldFourteen LAST SCOPE UPDATE}
 
         \vspace*{-1.5mm}  % blank line needed on previous line for vertical spacing
         \textcolor{rc-dark-green}{\rule{\textwidth}{0.4mm}}
 
         \vspace*{1mm}  % blank line needed on previous line for vertical spacing
-        {\RCSourceSansProSemiBoldEighteen \numprint{<<&calc.addresses_scanned>>}} \scalegraphicsnofig[1.2]{assets/<<#calc.addresses_scanned_up_flag>>grey-up-arrow<</calc.addresses_scanned_up_flag>><<#calc.addresses_scanned_down_flag>>grey-down-arrow<</calc.addresses_scanned_down_flag>><<#calc.addresses_scanned_flat_flag>>no-change-blue-arrows<</calc.addresses_scanned_flat_flag>>}
+        %{\RCSourceSansProSemiBoldEighteen \numprint{<<&calc.addresses_scanned>>}} \scalegraphicsnofig[1.2]{assets/<<#calc.addresses_scanned_up_flag>>grey-up-arrow<</calc.addresses_scanned_up_flag>><<#calc.addresses_scanned_down_flag>>grey-down-arrow<</calc.addresses_scanned_down_flag>><<#calc.addresses_scanned_flat_flag>>no-change-blue-arrows<</calc.addresses_scanned_flat_flag>>}
```

Comment:
Can we remove this line versus commenting it out?
```suggestion
```

Reply:
Please disregard this change. We will create a new request for this change.

Reply:
Does that mean you will revert or remove the commit?

Reply:
Please disregard this request and remove the comment tag to allow the report to look as it did before the request. Please call me if you need additional clarification.

Reply:
Please see commit a945299.

Reply:
That is good.